packages/toolkit/plugin/src/manager/shared.ts

```diff
@@ -1,4 +1,4 @@
-import { dagSort } from '../utils/pluginDagSort';
+import { pluginDagSort } from '@modern-js/utils';
```

Comment:
The `@modern-js/plugin` will be used at the browser side, so we should not import from the entry of `@modern-js/utils`.

We can move the `pluginDagSort` method to `@modern-js/utils/plugin-dag-sort`.